lib/proxyIntegration.js

```diff
@@ -7,6 +7,13 @@ const NO_MATCHING_ACTION = request => {
     }
 };
 
+function addCorsHeaders(toAdd) {
+    toAdd["Access-Control-Allow-Origin"] = "*";
+    toAdd["Access-Control-Allow-Methods"] = "GET,POST,PUT,DELETE,HEAD";
+    toAdd["Access-Control-Allow-Headers"] = "Content-Type,X-Amz-Date,Authorization,X-Api-Key,X-Amz-Security-Token";
+    return toAdd;
+}
+
 function process(proxyIntegrationConfig, event) {
```

Comment:
It is possible to cleanup the method 'process' a little bit?

Reply:
true... well that would be a small refactoring session.... not in this PR

Reply:
ok, lets do next week or so 👍